packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -151,7 +151,6 @@ export default function Keypad(props: Props) {
                 <View style={styles.keypadInnerContainer}>
                     <View
                         style={[styles.keypadGrid, gridStyle]}
-                        tabIndex={0}
                         aria-label="Keypad"
```

Comment:
With the `tabIndex` removed, is the `aria-label` still needed?

Reply:
Let me check and see how it sounds before removing the aria-label.

Reply:
Just curious - how did the screen reader test go with/without the `aria-label`?